fabric/build.gradle.kts

```diff
@@ -34,11 +35,13 @@ dependencies {
     api(project(":api"))
 
     // Compatibility with other mods
+    modCompileOnly("maven.modrinth:sodium:${property("sodium")}")
```

Comment:
Reason for having enableSodium and enableIris was to allow for snapshot development, Sodium doesn't have a public snapshot version out so I set things up to be able to compile the game without.

Reply:
To compile the sodium-targeting mixins you would either need to include the class at compile time or reference the class via a target string instead. I'll swap the class references out for target strings so you don't need a compile-time reference.

Reply:
nvm I see you have the conditions in the mixin plugin so they'll never run. I understand now

Reply:
Yeah, I set up something for gradle to exclude the Sodium files to make it function. No release should be built without Sodium there, this is just for testing purposes.